lib/main.dart

```diff
@@ -13,113 +16,294 @@ class MyApp extends StatelessWidget {
+  String myValue = '';
+  String holder = "";
   @override
   Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
     return Scaffold(
-      appBar: AppBar(
-        // TRY THIS: Try changing the color here to a specific color (to
-        // Colors.amber, perhaps?) and trigger a hot reload to see the AppBar
-        // change color while the other colors stay the same.
-        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
       body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          //
-          // TRY THIS: Invoke "debug painting" (choose the "Toggle Debug Paint"
-          // action in the IDE, or press "p" in the console), to see the
-          // wireframe for each widget.
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+        child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+          RichText(
+            text: const TextSpan(
+              style: TextStyle(
+                  fontSize: 80,
+                  fontWeight: FontWeight.w800,
+                  color: Colors.black),
+              children: [
+                TextSpan(
+                    text: 'Para',
+                    style: TextStyle(color: AppColors.primaryColor)),
+                TextSpan(
+                    text: 'gon',
+                    style: TextStyle(color: AppColors.defaultTextColor)),
+              ],
+            ),
+          ),
+          Container(
+              alignment: Alignment.topLeft,
+              margin: const EdgeInsets.only(top: 40),
+              padding: const EdgeInsets.only(left: 50),
+              child: const Text('Zaloguj sie',
```

Comment:
This is very similar to 'Zarejestruj się` from the main page. Consider extracting them